storage/onpremise/redis/variables.tf

```
@@ -13,6 +13,7 @@ variable "redis" {
     node_selector      = any
     image_pull_secrets = string
     max_memory         = string
+    max_memory_samples = optional(string, "")
```

No optionals in this repo:
```suggestion
    max_memory_samples = number
```